Resources/ServerInfo/Guidebook/ServerRules/SiliconRules/RuleS8DefaultCrewDefinition.xml

```diff
@@ -1,4 +1,15 @@
 ﻿<Document>
   # Silicon Rule 8 - Your HUD determines who is crew
   Unless a law redefines the definition of crew, then anyone who the HUD indicates to you has a job, including passengers, is a crewmember. You cannot do something that causes someone to not be considered crew, but you can allow someone else to do something that causes someone to not be crew.
+  
+  The following icons are not considered crew:
+  <Box>
+    <GuideEntityEmbed Entity="NoIdIcon" Caption="No ID"/>
+    <GuideEntityEmbed Entity="UnknownIcon" Caption="Unknown"/>
+    <GuideEntityEmbed Entity="VisitorIcon" Caption="Visitor"/>
```

Comment:
I don't think there's truly a consensus on if visitors are crew (up to the Borgs to be consistent).

I find it weird that you're classifying visitors as non-crew, while they are protected under space law. Theoretically they have as much of a job as passengers.

Reply:
Visitors aren't considered crew because they are not part of station crew. It just means borgs do not have to listen to them but that does not mean they will murder them on sight 100% time either. Command is free to hire them to make them crew in silicon eyes or tell them to treat visitors as crew.

Reply:
CE and now Nik both have said that visitors are not crew.